docs/websockets/ewallet_api.md

```diff
@@ -0,0 +1,499 @@
+# eWallet API Websockets
+
+The eWallet API offers a websocket interface for real-time communication. Currently, it can only be used to receive events. If you're planning to use the transaction requests with the `require_confirmation`, websockets are the best way to handle the confirmations in a safe way.
+
+## Getting Started
+
+Before diving into the details of how websockets can be used, here is a sample flow. First, we initialize a socket connection and join a channel. Then, we wait for an event to be received before leaving the channel.
+
+1. Initialize socket connection by sending a request to `/api/socket`.
+
+2. Join a channel (`"transaction_request:some_id"`)
+
+Payload sent:
+```
+{
+  "topic": "transaction_request:some_id",
+  "event": "phx_join",
+  "ref": "1",
+  "data": {}
+}
+```
+
+Payload from server:
+```
```

Comment:
Can you use ```json to give some color to the snippet

`With``` `
```
{
  "success": true,
  "version": "1",
  "data": null,
  "error": null,
  "topic": "transaction_request:some_id",
  "event": "phx_reply",
  "ref": "1"
}
```


`With```json`
```json
{
  "success": true,
  "version": "1",
  "data": null,
  "error": null,
  "topic": "transaction_request:some_id",
  "event": "phx_reply",
  "ref": "1"
}
```